scripts/gen.py

```diff
@@ -99,6 +99,7 @@ class ArgTemplate(string.Template):
     'hamming_window',
     'hann_window',
     'empty_like',
+    'eye',
```

Comment:
Maybe we can alphabetically order the ones below the FIXME comment?

Reply:
fixed! ;)